src/analysis/retail/shaman/enhancement/modules/core/SpellUsable.tsx

```diff
@@ -13,6 +15,17 @@ class SpellUsable extends CoreSpellUsable {
 
     super.beginCooldown(triggeringEvent, spellId);
   }
+
+  isAvailable(spellId: number): boolean {
+    const hasAscendance = this.selectedCombatant.hasBuff(TALENTS.ASCENDANCE_ENHANCEMENT_TALENT.id);
+    if (spellId === SPELLS.WINDSTRIKE_CAST.id && !hasAscendance) {
+      return false;
+    }
+    if (spellId === TALENTS.STORMSTRIKE_TALENT.id && hasAscendance) {
+      return false;
+    }
+    return super.isAvailable(spellId);
+  }
```

Comment:
i *don't think* this impacts any of the spell availability metrics since they're checked at the end of the fight, but i'm not 100% on that.

it is technically correct and will produce possibly-useful debug logging so its fine to have.

Reply:
This shouldn't have made it's way in either, I'll take it out.